### ISSUE TITLE
Make the Message ID suffix customizable

### ISSUE THREAD

**Issue by phax** [phax/phase4#50]
By default all the message IDs end with "@phase4".
This should be extensible, so that @phase4.anyone" is possible as a suffix.
It needs to be ensured, that the rules of RFC 822 are maintained

**Comment by phax** [comment]
```
addr-spec   =  local-part "@" domain        ; global address
     
     local-part  =  word *("." word)             ; uninterpreted
                                                 ; case-preserved
     
     domain      =  sub-domain *("." sub-domain)
     
     sub-domain  =  domain-ref / domain-literal
     
     domain-ref  =  atom                         ; symbolic reference

     atom        =  1*<any CHAR except specials, SPACE and CTLs>

     specials    =  "(" / ")" / "<" / ">" / "@"  ; Must be in quoted-
                 /  "," / ";" / ":" / "\" / <">  ;  string, to use
                 /  "." / "[" / "]"              ;  within a word.
```

**Comment by phax** [comment]
Fixed in 1.1.1 - `MessageHelperMethods.setCustomMessageIDSuffix(String)`
